README.md

```diff
@@ -1,4 +1,4 @@
-# Housekeeping
+cleanup-threads# Housekeeping
```

Comment:
This doesn't look right ?

Reply:
<img width="1099" alt="Screen Shot 2019-04-08 at 11 26 25 AM" src="https://user-images.githubusercontent.com/74704/55717472-43bd9880-59f1-11e9-9825-23c6d42e53f1.png">